### ISSUE TITLE
Dragging nodes in Gephi >0.9.2 with forceatlas2/linlog mode will drag multiple nodes randomly across the graph

### ISSUE THREAD

**Issue by Farewe** [gephi/gephi#1876]
## Expected Behavior
When a single node is selected and dragged, only this node should be moving.

## Current Behavior
In Gephi 0.9.2 and 0.9.3 (I tried the last nightly build), when I try to drag a node, there are multiple nodes across the entire graph that are dragged. This happens after I used the forceatlas 2 algorithm with LinLog mode. These nodes do not necessarily have links and can be very distant from each other in the graph, with other nodes in-between that are not dragged.

## Possible Solution

This bug did not happen in Gephi 0.9.1 so this is what I am using for now.

## Steps to Reproduce

1. Load network (mine has ~14k nodes, ~59k links)
2. Spatialise with ForceAtlas2 and linlog mode
3. Try to drag nodes

## Your Environment

* Version used: Gephi 0.9.2 & 0.9.3 Nightly build
* Java version: 8 update 151
* Operating System: Win10 64

**Comment by Farewe** [comment]
Any update on this issue?

**Comment by eduramiba** [comment]
Not yet, sorry.

**Comment by Xzelan** [comment]
I'm having the same issue on MacOS Mojave.

**Comment by AsemioAaron** [comment]
I am also having this issue on MacOS Mojave.

**Comment by fgrogers** [comment]
same issue - any workaround ?

**Comment by Farewe** [comment]
As a workaround you can roll back to Gephi 0.9.1. I tend to switch between versions depending on whether or not I need to drag nodes manually on my graphs.

**Comment by ells** [comment]
Sorry to revive a relatively dead bug, but this is still quite persistent and makes the software nearly unusable for certain applications. 

It seems that the best solution, as mentioned by @Farewe above, is to maintain a separate instance of 0.9.1 for when you need to manually move nodes.

**Comment by Yakuzzo** [comment]
i deactivated the edges (navigator is next to the labeling on the bottom of the graph) to ne able to drag nodes seperately again:
gephi 0.9.2 / win10 64

**Comment by Yakuzzo** [comment]
> Thanks, @jacomyma can take a look. It seems weird that a dragging issue would only appear when using a specific layout, or layout option. Normally visualization and layout are decoupled. Are you sure this isn't just a coincidence?

you are right, it was a coincidence. it can change with a click on any navigator or switching between tabs.
